<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDlgDeviceExportPath</class>
 <widget class="QDialog" name="IDlgDeviceExportPath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select output path...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelHead">
     <property name="text">
      <string>Where to place..?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a sub-directory from above or enter a new one</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
